src/screens/userProfileScreens/EditUserProfileScreen.js

```diff
@@ -0,0 +1,141 @@
+import {
+	Alert,
+	Dimensions,
+	PixelRatio,
+	SafeAreaView,
+	ScrollView,
+	StyleSheet,
+	TextInput,
+	View
+} from "react-native";
+import PropTypes from "prop-types";
+import React, { useMemo, useState } from "react";
+import UserProfileAvatar from "../../components/profile/UserProfileAvatar";
+import { useDispatch, useSelector } from "react-redux";
+import GlobalStyles from "../../utils/GlobalStyles";
+import { COLOR_PALETTE } from "../../utils/constants";
+import Button, { BUTTON_TYPES } from "../../components/common/Button";
+import { launchImageLibrary } from "react-native-image-picker";
+import {
+	getUpdatedUserInformation,
+	handleUpdateProfile
+} from "../../utils/firebase";
+
+const width = Dimensions.get("window").width;
+
+const EditUserProfileScreen = ({ navigation }) => {
+	const dispatch = useDispatch();
+	const displayName = useSelector(
+		({ UserSlice }) => UserSlice?.userInformation?.displayName
+	);
+
+	const photoURL = useSelector(
+		({ UserSlice }) => UserSlice?.userInformation?.photoURL
+	);
+
+	const [editUserName, setEditUsername] = useState(
+		displayName || "Default User"
+	);
+	const [imagePreview, setImagePreview] = useState(photoURL || "");
```

Comment:
why does this have to be photoURL or empty string?

Reply:
When the user first creates account everything is set to default. So If there is no photoURL available (meaining they haven't already updatred their profile once) then the image will show the empty image icon instead